src/PagerEvent.js

```diff
@@ -76,7 +76,7 @@ const Unit = ({ code, dateTime, cancelled }) => {
   } else {
     classes.push('UNIT_15');
   }
-  return <span className={classes.join(' ')}>{code} </span>;
```

Comment:
Might be cleaner to just delete the space here and leave the parent component to decide formatting (as in, add the space)